Shared/Samples/Update related features/UpdateRelatedFeaturesView.swift

```diff
@@ -0,0 +1,294 @@
+    }
+    
+    /// A view displaying callout content, including editable "Annual Visitors" values.
+    /// Includes a picker to allow updating the selected visitor range.
+    var calloutContent: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("\(model.parkName)")
+                .font(.headline)
+            if !model.attributeValue.isEmpty {
+                Text("Annual Visitors:")
+                // Picker to allow the user to update visitor range.
+                Picker("Annual Visitors", selection: $model.selectedVisitorValue) {
+                    ForEach(model.visitorOptions, id: \.self) { option in
+                        Text(option).tag(option)
+                    }
+                }
+                .pickerStyle(.menu)
+                .onChange(of: model.selectedVisitorValue) { _, newValue in
+                    Task {
+                        do {
+                            try await self.model.setSelectedFeatureUpdate(newValue)
+                        } catch {
+                            self.error = error
+                        }
+                    }
+                }
+            }
+        }
+        .padding()
+    }
+    
+    /// The loading indicator overlay shown during data fetches.
+    var loadingView: some View {
+        ProgressView(
+               """
+               Fetching
+               data
+               """
+        )
+        .padding()
+        .background(.ultraThinMaterial)
+        .clipShape(.rect(cornerRadius: 10))
+        .shadow(radius: 50)
+        .multilineTextAlignment(.center)
+    }
+}
+
+extension UpdateRelatedFeaturesView {
+    @MainActor
+    @Observable
+    class Model {
+        /// A map with a topographic basemap style.
+        @ObservationIgnored var map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// A boolean value the reflects whether the callout should be shown or not.
+        var calloutVisible = false
+        
+        /// Feature layers and tables for parks and preserves
+        var parksFeatureLayer: FeatureLayer?
+        
+        /// Used for querying park features and related data.
+        var parksFeatureTable: ServiceFeatureTable?
+        
+        /// The service feature table representing preserve data.
+        /// Used to access and update related preserve features connected to parks.
+        var preservesTable: ServiceFeatureTable?
+        
+        /// Selected features and related features.
+        var selectedFeature: ArcGISFeature?
+        var relatedSelectedFeature: ArcGISFeature?
+        
+        /// The location of the callout on the map.
+        var calloutPlacement: CalloutPlacement?
+        
+        /// The screen (pixel) coordinates of the user's tap on the map view.
+        var screenPoint: CGPoint?
+        
+        /// The geographic `Point` corresponding to the user's tap location on the map.
+        /// This is used to center the map or position the callout.
```

Comment:
```suggestion
        /// The map point where the map was tapped.
```
This can be simplified.